elasticsearch/Dockerfile

```diff
@@ -45,7 +45,7 @@ ADD index_patterns/ ${ES_HOME}/index_patterns/
 ADD init/ ${ES_HOME}/init/
 ADD kibana_ui_objects/ ${ES_HOME}/kibana_ui_objects/
 ADD probe/ ${ES_HOME}/probe/
-ADD init.sh run.sh prep-install.${RELEASE_STREAM} install.sh ${HOME}/
+ADD init.sh run.sh prep-install* install.sh ${HOME}/
```

Comment:
Do we actually use the `prep-install.prod` file at all? If not, can we simplify the ADD directive to just copy `prep-install.origin`?

Reply:
> Do we actually use the `prep-install.prod` file at all? If not, can we simplify the ADD directive to just copy `prep-install.origin`?

Yes.  they get used differently depending if you are building in CI or vi ART.  It's an attempt to provide normalization between the two build options.